streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/service/StreamCatalogService.java

```diff
@@ -2341,4 +2343,55 @@ private String getTopologyComponentBundleJarName (TopologyComponentBundle topolo
     private Collection<File> listFiles(List<QueryParam> queryParams) {
         return dao.find(File.NAME_SPACE, queryParams);
     }
+
+    public Collection<TopologyTestRunHistory> listTopologyTestRunHistory(Long topologyId) {
```

Comment:
May be i did not put it clearly. I meant should we be providing an api to return results for a particular version of a topology? Presumption is they could be running multiple tests with same version of topology. Let me know if i am misunderstanding something.